pkg/controller/clone-controller.go

```diff
@@ -0,0 +1,264 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+	//"encoding/json"  temporary for debug
+	"github.com/golang/glog"
+	. "github.com/kubevirt/containerized-data-importer/pkg/common"
+	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+)
+
+const (
+	// pvc annotations
+	AnnCloneRequest    = "k8s.io/CloneRequest"
```

Comment:
I think you should not use the kubernetes prefix here and below. Better to keep this related to kubevirt and use the _cdi.kubevirt.io_ anno prefix.  Also better if all annotation use same prefix.

Reply:
This is from Trello card. @aglitke ?

Reply:
Actually please keep the current annotations.  We want to share the same annotation that external provisioners will use.  That's how we achieve interoperability between smart-clone and host-assisted cloning.

Reply:
I think you're right but we already have two provisioners using the current k8s.io/CloneRequest so I'd like to keep that for now for interoperability.  We can change it later when we adapt to other design changes requested by the community.